sdk/core/core-sse/package.json

```diff
@@ -73,28 +69,37 @@
     "lint:fix": "eslint package.json api-extractor.json src --ext .ts --ext .cts --ext .mts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json api-extractor.json src --ext .ts --ext .cts --ext .mts",
     "pack": "npm pack 2>&1",
+    "start-server": "tsx test/server/start.mts",
     "test:browser": "npm run clean && npm run build && npm run integration-test:browser",
     "test:node": "npm run clean && tshy && npm run integration-test:node",
     "test": "npm run clean && tshy && npm run unit-test:node && dev-tool run build-test && npm run unit-test:browser && npm run integration-test",
     "unit-test:browser": "npm run clean && tshy && dev-tool run build-test && dev-tool run test:vitest --no-test-proxy --browser",
-    "unit-test:node": "dev-tool run test:vitest --no-test-proxy",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+    "unit-test:node": " start-server-and-test start-server tcp:3000 vitest:node",
```

Comment:
Wonder if it would be possible to drop the dev dependency by starting and stopping the server programmatically in a `beforeAll`/`afterAll`?

Alternatively, maybe we could use `concurrently` instead of `start-server-and-test`, since we already depend on that in e.g. [dev-tool](https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/dev-tool/package.json#L54).

Reply:
The first suggestion adds syntactic overhead for every suite that uses it. Granted, it is a single suite for now but I always try to not open up room for such overheads. 

For the second suggestion, they're too different packages. `start-server-and-test` doesn't start the testing before the server is ready to accept requests. It is also used in event-hubs anyway.

Reply:
Ah, I didn't realize we were already using this somewhere else. Then I have no issue.

It does seem like vitest provides a 'global setup' feature though which seems like it would be  well suited for this purpose: https://vitest.dev/config/#globalsetup

Reply:
nice, I like global setup here because I can now get rid of duplicating the port in two places, addressed in https://github.com/Azure/azure-sdk-for-js/pull/30707/commits/d9a8ea5764e27702ef7c89a621b0373e189e63dc